### PR TITLE
STIPS 2.1 Pull Request

### PR DESCRIPTION
**Pull request by gmzsebastian**
- Fixed generic bugs
- Updated background to use Pandeia
- Fixed flux values to correct zeropoints

### CODE REVIEW

**Comment by gmzsebastian** [comment]
Additional changes:
- Updated the code to make sure it works with the latest version of Pandeia, 3.0
- Updated the documentation to make sure all the new changes are reflected properly, and that they match what RDox says.

**Review by ojustino — COMMENTED** [review]
Hi @gmzsebastian, thanks for your work on this release and your patience with the review. I left a few review comments on specific lines of code and have included some larger questions as a checklist here. I'll update the list as each line is resolved.

- [ ] [I have opened a PR](https://github.com/gmzsebastian/STScI-STIPS/pull/3) against your branch that will a) change bullet characters back to normal in the docs after `stsci_rtd_theme` fixed [an issue that led us to change them](https://github.com/spacetelescope/STScI-STIPS/pull/163/#pullrequestreview-1081339571) during the 2.0.0 release, and b) update the package version in `setup.cfg` and `stips/__init__.py` to match our intended release number. If you approve, please accept and merge it into this PR branch.

- [x] Are you able to fully run the "Basic" and "Advanced I" tutorial notebooks while off the VPN? While both are OK for me on the VPN, if I'm not connected, cells that feature calls to `ObservationModule()` trigger a warning saying "Vega spectrum from /grp/hst/cdbs//calspec/alpha_lyr_stis_010.fits" is missing and an error saying that the `COMPFILES` list in the `RomanInstrument` class is empty.

- [x] An extra touch to the "Advanced II" notebook that I think would be helpful is for the tick labels of the 60x60-pixel cutout region to match their corresponding pixel numbers in the original FITS image. If you agree, you can adapt the following code for the cells that need it in the notebook:

```
# add this import to the first cell
# (and alphabetize all of them as indicated in PR code review comments)
import matplotlib.ticker as ticker

...

# in cutout plots, try the following:
x_mid = 88
y_mid = 88
half = 30

# to plot cutout region:
fig0, ax0 = plt.subplots(figsize=(4,4))
ax0.imshow(test_psf[x_mid - half:x_mid + half,
                    y_mid - half:y_mid + half])

# create custom formatter for cutout region axes (run once)
ticks_x = ticker.FuncFormatter(lambda x, pos: f"{x + x_mid - half:.0f}")
ticks_y = ticker.FuncFormatter(lambda y, pos: f"{y + y_mid - half:.0f}")

# replace default tick labels with cutout pixel indices
# (run in every cell that shows a plot of the cutout region)
ax0.set_xticks([x for x in ax0.get_xticks() if 0 <= x < half*2])
ax0.xaxis.set_major_formatter(ticks_x)
ax0.yaxis.set_major_formatter(ticks_y)
```

- [ ] What do you think of putting `psf_WFI_2.0.0_F129_sca01.fits` into a "notebook_data" directory or something similar so `notebooks/` contains only that folder and the tutorial notebooks? If we go this route, I don't modifying the notebooks so the data files they generate also go in that folder.

- [ ] Since you updated the name of the Basic Tutorial notebook, we need to ensure that other pages that reference it don't end up with broken links. One example is the end of the introductory paragraph in the RDox "STIPS Basic Use Tutorial." Can you think of any others? 

- [x] Since `stips/utilities/tests/test_makePSF.py` is new and `stips/instruments/wfi.py` has almost been entirely remade, please do a PEP8 code style check on these files and commit the edits. Other places in STIPS are not strictly PEP8-compliant, but we should at least endeavor to follow the style guide when we introduce new code. If you're not already familiar with how to do this, I have used [the flake8 Python package](https://pypi.org/project/flake8/) and can give support it if needed.

File: stips/instruments/instrument.py [hunk 1]
```diff
@@ -82,7 +82,8 @@ def __init__(self, **kwargs):
         self.DETECTOR_OFFSETS = self.DETECTOR_OFFSETS[:n_detectors]
         self.OFFSET_NAMES = self.OFFSET_NAMES[:n_detectors]
         if hasattr(self, "N_OFFSET"):
-            self.CENTRAL_OFFSET = self.N_OFFSET[n_detectors]
+            # Set the central offset to SCA01
+            self.CENTRAL_OFFSET = self.N_OFFSET[1]
```
[the remark this answers is not in the document]
Comment (gmzsebastian) [reply stips/instruments/instrument.py:86]: The offsets are defined in terms of SCA01, so that is necessary.

**Comment by gmzsebastian** [comment]
I've gone through all the updates and suggestions for the STIPS 2.1 code review. I'll do some final tests and after that we should be ready for release!

**Comment by gmzsebastian** [comment]
Accidentally merged PR167 into Master

**Pull request merged**